src/components/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,52 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const baseUrl = 'https://api.spacexdata.com/v3/rockets';
+const initialState = {
+  rockets: [],
+  isLoading: false,
+  errorMessage: '',
+};
+
+export const fetchRockets = createAsyncThunk('rockets/fetchRockets', () => axios.get(baseUrl).then((response) => response.data));
```

Comment:
- [ ] Kindly filter the data to be saved from the API. It's not all items from the API that should be stored. Kindly visit [this requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store) and see the items to be saved.